src/main/src/main/java/org/geoserver/catalog/FeatureTypeInfo.java

```diff
@@ -153,4 +153,23 @@
     boolean isCircularArcPresent();
 
     void setCircularArcPresent(boolean arcsPresent);
+
+    /**
+     * Controls if coordinates measures should be included in WFS outputs.
+     *
+     * @return TRUE if measures should be encoded, otherwise FALSE
+     */
+    default boolean getEncodeCoordinatesMeasures() {
```

Comment:
Should be "isEncodeCoordinateMeasures" or even better "isCoordinateMeasureEncoded"

Reply:
I agree :stuck_out_tongue:, done.